apiserver/facades/client/applicationoffers/applicationoffers_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 type applicationOffersSuite struct {
 	baseSuite
-	api *applicationoffers.OffersAPI
```

Comment:
Since facade v1 is still around, I do not think we'd want to lose v1 tests...

Reply:
Agreed. Usually we keep all old tests for previous version and add new tests for the calls affected in new version.